tutorials/chartjs/label-color.md

```diff
@@ -0,0 +1,119 @@
+With chartjs version 3.7.1, you can change the color of the labels.
```

Comment:
ChartJS

Reply:
Also, add "by setting the `scales.x.ticks.color` and `scales.y.ticks.color` options"